projects/mysadna2/src/app/user-details/user-details.component.html

```diff
@@ -1,34 +1,45 @@
 <div *ngIf="user">
-  <h2>{{user.firstName}} {{user.lastName}}</h2>
-  <span>
-    <div class="userName">{{user.userName}}</div>
-    <div class="email">{{user.email}}</div>
-    <ul>
-      <li *ngFor="let socialNetwork of user.socialNetworks">
-        {{socialNetwork}}
-      </li>
-    </ul>
-  </span>
-  <span>
-    <div class="projects">
-      <h4>Projects:</h4>
-      <ul>
-        <li *ngFor="let project of userProjects">
-          <a [routerLink]="'/project-details/' + project.projectId">{{project.projectName}}</a>
+  <h2 class="userHeader">{{user.firstName}} {{user.lastName}}</h2>
+  <div class="userContainer">
+    <div class="leftContainer">
+      <img src="{{user.userImg}}" class="userImage">
+      <div class="userName">{{user.userName}}</div>
+      <div class="email">{{user.email}}</div>
+      <ul class="social-Networks">
+        <li *ngFor="let socialNetwork of user.socialNetworks" class="socialNetwork">
+          <div class="{{socialNetwork}}">
+            <!-- svg -->
+            <img src="./assets/images/{{socialNetwork}}.svg" class="{{socialNetwork}}">
+            <!-- png -->
+            <!--<img src="./assets/images/linkedin.png" srcset="./assets/images/linkedin@2x.png 2x,./assets/images/linkedin@3x.png 3x" class="linkedin">-->
```

Comment:
Hardcoded code?

Reply:
Yes.

Reply:
I didn't know what the png option meant, so I left it there in a comment for the moment.